<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vente</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">*{
border : none;
}

#widget{
background-color: rgb(29, 35, 50)
}

QScrollArea #scrollAreaWidgetContents {
background-color: rgba(29, 33, 50, 45);
border: 4px solid red
}

QTabBar::tab {
background-color: rgb(227, 227, 227);
width: 120px;
height: 55px;
padding-left: 10px;
padding-right : 10px;
border-top-left-radius: 5px;
border-top-right-radius: 5px; 
font: 63 18pt &quot;Bahnschrift SemiBold Condensed&quot;;
margin-right: 5px
}

#tab{
background-color: rgb(227, 227, 227)
}

QPushButton, QLineEdit, QSpinBox, QDoubleSpinBox, QPlainTextEdit{
padding: 8px;
background-color: white;
color: rgb(29, 35, 50);
border-radius: 5px;
}

QLineEdit, QSpinBox, QDoubleSpinBox, QPlainTextEdit{
border: 1px solid rgb(29, 35, 50);
}

QPushButton{
color: white
}

#enregistrerButton{
background-color: rgb(30, 255, 30);

}

#annulerButton{
background-color : rgb(255, 29, 29)
}

#imageLabel{
border: 3px dashed rgb(29, 35, 50);
}

#addClientPushButton{
color:black;
border: 1px solid rgb(29, 35, 50);
}


</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <widget class="QWidget" name="tab">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="icon">
          <iconset>
           <normaloff>../pictures/requisition.png</normaloff>../pictures/requisition.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Vente</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>501</width>
                  <height>874</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,0">
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,20">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QWidget" name="widget_8" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_14">
                         <property name="font">
                          <font>
                           <family>Eras Demi ITC</family>
                           <pointsize>13</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Categorie :</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="categorieLabel">
                         <property name="font">
                          <font>
                           <family>Eras Bold ITC</family>
                           <pointsize>14</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Braclets</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QWidget" name="widget_7" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_12">
                         <property name="font">
                          <font>
                           <family>Eras Demi ITC</family>
                           <pointsize>13</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Nom Produit :</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="nomProduitLabel">
                         <property name="font">
                          <font>
                           <family>Eras Bold ITC</family>
                           <pointsize>14</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Green Braclet </string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_6" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>250</height>
                       </size>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_3">
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QLabel" name="imageLabel">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>200</width>
                           <height>200</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>../pictures/photo.jpg</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="3" column="1">
                     <widget class="QLineEdit" name="phoneNumberLineEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="label_21">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Address :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLineEdit" name="addressLineEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="1">
                     <widget class="QLineEdit" name="emailLineEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="0">
                     <widget class="QLabel" name="label_24">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Notes :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="1">
                     <widget class="QLineEdit" name="dateEntreeLineEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="readOnly">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="0">
                     <widget class="QLabel" name="label_23">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Date d'Entrée :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="1">
                     <widget class="QPlainTextEdit" name="plainTextEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Quantité Vendue :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QDoubleSpinBox" name="prixVendueDoubleSpinBox">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="decimals">
                       <number>3</number>
                      </property>
                      <property name="maximum">
                       <double>999999.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="costumerlineEdit">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QSpinBox" name="quantiteVendueSpinBox">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="maximum">
                       <number>999999</number>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QPushButton" name="addClientPushButton">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="cursor">
                       <cursorShape>PointingHandCursor</cursorShape>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_9">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Client :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_8">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Prix De Vente :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_22">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Phone :</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="0">
                     <widget class="QLabel" name="label_13">
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Email :</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="enregistrerButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Enregistrer</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="annulerButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Annuler</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../interface_images.qrc"/>
 </resources>
 <connections/>
</ui>
